VehicleCode/Input/DataCollector.cpp

```diff
@@ -58,8 +125,33 @@ void DataCollector::checkSensors() {
 void DataCollector::readData(Sensor* sensor) {
     // Call the readInputs method to obtain an array of ints
     rawData = sensor->readInputs();
-    Serial.println(rawData);
-    
+
+    if(sensor->getId() == SWITCH_ID && front) { // Switch Checks
```

Comment:
I didn't check the logic here but it should be similar to what used to be in car class